wagtail_wordpress_import/test/fixtures/raw_xml.xml

```diff
@@ -170,6 +172,8 @@ Nihil hic munitissimus habendi senatus locus, nihil horum?.&lt;/blockquote&gt;</
             <wp:post_type>page</wp:post_type>
             <wp:post_password/>
             <wp:is_sticky>0</wp:is_sticky>
+            <category domain="category" nicename="blogging">Cars</category>
+            <category domain="category" nicename="life">Computing</category>
```

Comment:
**Comment**: The `nicename` attributes don't correspond to the displayed text.

Reply:
Thanks I will correct it.